README.md

```diff
@@ -464,6 +464,24 @@ GoTrue exposes the following endpoints:
   This will revoke all refresh tokens for the user. Remember that the JWT tokens
   will still be valid for stateless auth until they expires.
 
+## Testing
+
+Testing requires a few things.
+- docker installed on the machine for the mysql image
+- [pop](https://github.com/markbates/pop) and soda installed
+
+The necessary commands are:
+```
+go get github.com/markbates/pop/...
+go install github.com/markbates/pop/soda
```

Comment:
these go get should probably be included in the `make deps` target, or we could have another target like `make dev_deps`.

Reply:
hmmm actually those are there - but they didn't work 
```
 deps: ## Install dependencies.
   @go get -u github.com/markbates/pop/soda
   @go get -u github.com/golang/lint/golint
   @go get -u github.com/Masterminds/glide && glide install
```

Reply:
removing for the time being